try {
  throw new Error();
} catch (e) {
  console.error("Oh No");
}
